libraries/mapr_utils.rb

```diff
@@ -18,6 +18,19 @@ def configured_disks
       disks.sort
     end
 
+    def unpartitioned_disks(node)
+      disks_and_partitions = ::Dir.glob ['/dev/disk/by-id/wwn-*']
+      partition_regex = Regexp.new(/-part[0-9]*/)
+      partitionned_disks = disks_and_partitions.grep(partition_regex).map{|dev| dev.sub(partition_regex, '')}
```

Comment:
I think is better to add **uniq** here to remove duplicate entries

Reply:
not necessary but done